### PR TITLE
Add work page

### PR DESCRIPTION
**Pull request by cobbie**
## Description
- Created the general UI for the Work webpage
- To follow UI: footer, text on images under Work/Case Studies section of page, adjusting of dimensions
- To follow Webpage overall: Responsiveness, functionality/interactivity

## Pull Request Changes
- State/stateless components per section of page created
- General HTML/CSS layout done but a lot of refining to be done

## Screenshots (optional)
screen shot 2018-10-14 at 9 07 53 pm
screen shot 2018-10-14 at 9 08 05 pm
screen shot 2018-10-14 at 9 08 17 pm
screen shot 2018-10-14 at 9 08 26 pm
screen shot 2018-10-14 at 9 08 33 pm

### CODE REVIEW

File: src/components/state/Headers/header.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import communityLanding from '../../../resources/img/community-landing.svg'
+import './style.css'
+import mainImage from '../../../resources/img/work-imgs/productivity.svg'
+
+class Header extends Component {
+    state = {  }
+    render() { 
+        return ( 
+            <div>
+                <div className = 'headers'>
```
Comment (xaviablaza) [inline src/components/state/Headers/header.jsx]: Use double quotes here.

File: src/components/state/Headers/header.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                    <h4 className="workwithux">WORK WITH UX SOCIETY</h4>
```
Comment (xaviablaza) [inline src/components/state/Headers/header.jsx]: Use the `section-label` class here.

File: src/components/state/Headers/header.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    <h1 className = "mainheader">Crafting the best experiences <br />to achieve your goals</h1>
+                    <div className="buttonDiv"><button className="ContactUs">CONTACT US</button></div>
```
Comment (xaviablaza) [inline src/components/state/Headers/header.jsx]: Use the `journey-button` class in the button tag.

File: src/components/state/Process/process.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import arrowRight from '../../../resources/img/work-imgs/arrowRight.svg'
+import arrowLeft from '../../../resources/img/work-imgs/arrowLeft.svg'
+class Process extends Component {
+    state = {  }
+    render() { 
+        return ( 
+        <div className = "process">
+            <div className="headers">
+                <h4 className="processMiniHeader">PROCESS</h4>
+                <h1 className = "methodMadnessHeader">The method to our madness</h1>
```
Comment (xaviablaza) [inline src/components/state/Process/process.jsx]: This isn't centered: 
image

File: src/components/state/Process/process.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                <img src={productivity2} className="processImage"></img>
+                <img src={arrowRight} className = "arrowRight"></img>
+                <img src={arrowLeft} className = "arrowLeft"></img>
+                <h2 className="discovery">01. Discovery</h2>
+                <p className="text">This is our research phase. We try to get as much information about
```
Comment (xaviablaza) [inline src/components/state/Process/process.jsx]: Use Proxima Nova for your text -- you can find the webfonts here: https://github.com/elliakou/proxima-nova-web-fonts

File: src/components/stateless/Services/services.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import './style.css'
+import productivityGirl from '../../../resources/img/work-imgs/productivity-2.svg'
+
+const Services = () => {
+return (
+    <div className = "mainServicesComp">
+                <div className= "services">
+                    <h4>SERVICES</h4>
+                    <h2>What we can do for you</h2>
```
Comment (xaviablaza) [inline src/components/stateless/Services/services.jsx]: Overlapping text on wider resolutions: 
image

File: src/components/state/Work/Work.jsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    state = {  }
+    render() { 
+        return ( 
+            <div className = "mainWork">
+                <div className = "workHeaders">
+                    <h4 className = "workHead">WORK</h4>
+                    <h1 className = "caseStudies">Case Studies</h1>
+                </div>
+                <div className="tedxDiv">
+                    <img src = {tedx}></img>
```
Comment (xaviablaza) [inline src/components/state/Work/Work.jsx]: These cards could be separate components. You can make the text and the href for these cards props that are passed to the component. 
image

File: src/components/stateless/Logos/Logos.jsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+                    </div>
+                    <div className="indivLogo lions">
+                    <img src={lions}></img>
+
+                    </div>
+                    <div className="indivLogo uxph">
+                    <img src={uxph}></img>
+
+                    </div>
```
Comment (xaviablaza) [inline src/components/stateless/Logos/Logos.jsx]: The images here should be props. The icons are not centered on wider resolutions: 
image

File: src/components/state/Process/process.jsx (hunk 5 above)
Comment (cobbie) [reply src/components/state/Process/process.jsx]: Changed it to Montserrat temporarily as was discussed in the Slack convo, will wait for further instruction 👍

File: src/components/state/Headers/header.jsx (hunk 2 above)
Comment (cobbie) [reply src/components/state/Headers/header.jsx]: For the main div that encompasses the whole component or for the current "headers" div?

Comment (xaviablaza) [reply src/components/state/Headers/header.jsx]: On line 12, the h4 tag.

**Comment by xaviablaza** [comment]
@cobbie Just request a review from me or Harvey when ready to merge :+1:

**Pull request merged**